Input/input.css

```diff
@@ -0,0 +1,37 @@
+@import '../style.css';
+
+.float-container {
+    border: solid 2px var(--shadow-color);
+    position: relative;
+    padding: 0 8px;
```

Comment:
Better to change px to rem.

Reply:
As suggested by vikram in the session if anything is upto 10px we can keep it in px